src/CSharpMongoMigrations.Demo/Runner.cs

```diff
@@ -5,9 +5,11 @@ namespace CSharpMongoMigrations.Demo
     public class Runner
     {
         public static void Main(string[] args)
-        {            
-            var runner = new MigrationRunner("mongodb://localhost:27017/TestMigrations", Assembly.GetExecutingAssembly().FullName, new MigrationFactory());
-            
+        {
```

Comment:
No need to expand the statement. It just brings useless line of code into the PR. 
Could you please rollback it.

Reply:
Line break removed. Few EOL spaces got removed as well.